core/controllers/custom_landing_pages.py

```diff
@@ -30,15 +30,27 @@ class FractionLandingRedirectPage(base.BaseHandler):
     @acl_decorators.open_access
     def get(self):
         """Handles GET requests."""
-        self.redirect('/learn/maths/fractions')
+        self.redirect('/maths/fractions')
```

Comment:
math or maths? Ditto elsewhere. Seems to be some inconsistency, you might want to check with Danial.

Reply:
I based the following thought process off of our discussion [here](https://docs.google.com/document/d/1x5ICyvrxPHBJFLYmlhINvwmPcLEVFLGCbteaNRIAPL0/edit?disco=AAAAGapcdkI).
The thing is, the classroom name is 'Math' and the subject name is 'maths'. Now, as per your suggestion (URLs are usually lowercased), I've currently changed the `/classroom/Math` URL to `/math` and the topic URL is changed to `/<subject>/<topic>/`(general form), which is why since the subject name is 'maths', it's being displayed as `/maths/fractions`. 
That said, if we'd like to have this as `/math/fractions`, I can change the subject name here and everywhere else from 'maths' to 'math'. 

_Side note:_ I guess this might have been named this way since we usually say 'Math classroom', not 'Maths classroom', and we call the subject maths since it short for mathematics. It's just a guess, though.

Reply:
For that discussion, we only got to the point of "don't support both". But I think you might want to check with Danial about whether the two need to be consistent. I think "maths" is more English and "math" is more American, and more of our target audience is familiar with the former system, but I don't know how much this matters.